oshi-core/src/main/java/oshi/software/os/linux/LinuxOperatingSystem.java

```diff
@@ -247,9 +247,13 @@ public OSProcess getProcess(int pid) {
         proc.setPriority(ParseUtil.parseIntOrDefault(split[17], 0));
         proc.setVirtualSize(ParseUtil.parseLongOrDefault(split[22], 0L));
         proc.setResidentSetSize(ParseUtil.parseLongOrDefault(split[23], 0L) * this.memoryPageSize);
-        proc.setKernelTime(ParseUtil.parseLongOrDefault(split[14], 0L) * 1000L / hz);
-        proc.setUserTime(ParseUtil.parseLongOrDefault(split[13], 0L) * 1000L / hz);
-        proc.setStartTime(bootTime + ParseUtil.parseLongOrDefault(split[21], 0L) * 1000L / hz);
+        if(hz != 0) {
+            proc.setKernelTime(ParseUtil.parseLongOrDefault(split[14], 0L) * 1000L / hz);
+            proc.setUserTime(ParseUtil.parseLongOrDefault(split[13], 0L) * 1000L / hz);
+            proc.setStartTime(bootTime + ParseUtil.parseLongOrDefault(split[21], 0L) * 1000L / hz);
+        } else {

```

Comment:
Actually, on second thought.... if hz == 0 we should explicitly re-set it to its default value of 1000L.

Reply:
Easily done! PR updated!